test/date_types.test.js

```diff
@@ -232,7 +232,7 @@ describe('MySQL DATE, DATETTIME, TIMESTAMP types on server with non local TZ (+0
 });
 
 var prepareModel = function(tz, done) {
-  db = getSchema({timezone: tz});
+  db = getSchema({timezone: tz, legacyDateTypeProcessing:false});
```

Comment:
default was 'undefined' or true. these tests were on fixed date types processing. but it is not mater now, because #257 was reverted. :(